{"errors":{"name":"Bucket with given name already exists"},"summaries":{"ramSummary":{"total":268435456,"otherBuckets":134217728,"nodesCount":1,"perNodeMegs":128,"thisAlloc":134217728,"thisUsed":0,"free":0},"hddSummary":{"total":43848368128,"otherData":40336229569,"otherBuckets":4269108,"thisUsed":0,"free":3507869451}}}